dist_tplink_v3/install-common.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+install() {
+    if [[ -z "${ADB}" ]]; then
+        echo "\$ADB not set, did you run this from install-linux.sh or install-mac.sh?"
```

Comment:
We should make this match our unified install script format for mac and linux `install.sh` as we have for the orbic. Also I notice there is not mac install script, will you add that or have you tested it at all?